buildrunner/config/validation.py

```diff
@@ -253,9 +256,13 @@ def validate_multiplatform_build(
             if not isinstance(step.build.platforms, list):
                 raise ValueError(f"platforms must be a list in build step {step_name}")
 
-            if step.build.cache_from:
+            if (
+                step.build.platforms
+                and isinstance(step.build.cache_from, list)
+                and all(isinstance(x, str) for x in step.build.cache_from)
+            ):
                 raise ValueError(
-                    f"cache_from is not allowed in multi-platform build step {step_name}"
+                    f"{BUILD_MP_CACHE_ERROR_MESSAGE} {step_name} cannot be a list(str) for multiplatform images {type(step.build.cache_from)}]"
```

Comment:
The last `type` call here will just show something nonsensical to the user, can you can add parenthesis or something to make it clear it's for debug purposes?

Reply:
Removed that last part of the message.